ucm2/MediaTek/mt8390-evk/HiFi.conf

```diff
@@ -0,0 +1,217 @@
+SectionVerb {
+	EnableSequence [
+		cset "name='HDMI_OUT_MUX' Connect"
+		cset "name='DPTX_OUT_MUX' Disconnect"
+		cset "name='O000 I000 Switch' on"
```

Comment:
I don't think that all those controls must be touched in the verb init section. See mt8365-evk as example. Is any benefit with turning the controls off ? Power savings?

Reply:
Agree, I will remove the verb init section.

Reply:
Done